docs/upgrade/focal_prep.rst

```diff
@@ -0,0 +1,179 @@
+Ubuntu 20.04 LTS (Focal) migration - Preparatory steps
+======================================================
+On April 30, 2021, Ubuntu 16.04 LTS (Xenial) will reach End of Life. After this
+date, no new security updates to the base operating system will be provided.
+It is therefore of critical importance for the security of all SecureDrop
+instances to upgrade to Ubuntu 20.04 LTS (Focal) before **April 30**.
+
+.. important::
+
+   For security reasons, the *Source Interface* will automatically be
+   disabled on SecureDrop servers running Ubuntu 16.04 after April 30, 2021.
+
+Support for Ubuntu 20.04 LTS (which will receive security updates until
+April 30, 2025) will be included in SecureDrop 1.8.0, scheduled to
+be released on March 2, 2021. Please do **not** attempt to upgrade to Ubuntu 20.04
+before then. The migration to Ubuntu 20.04 will require a reinstall from a
+backup.
+
+We recommend that you plan a two day maintenance window
+**between March 9 and April 30** to back up your instance, perform the migration,
+and test your instance once it is migrated.
```

Comment:
Why March 9? That gives us a week to notice any "early bird" issues, and a tiny bit of slack in case we must reschedule the 1.8.0 release. We did the same thing for the Xenial migration.